sdcm/kcl_thread.py

```diff
@@ -45,7 +45,7 @@ def build_stress_cmd(self):
         return stress_cmd
 
     def _run_stress(self, loader, loader_idx, cpu_idx):
-        docker = RemoteDocker(loader, "scylladb/hydra-loaders:kcl-jdk8-20210215",
+        docker = RemoteDocker(loader, "scylladb/hydra-loaders:kcl-jdk8-20210526-ShardSyncStrategyType-PERIODIC",
```

Comment:
Please remove this part from it.

Reply:
@dkropachev why removing?

p.s. - this PR may still be on hold.

Reply:
@yarongilor , my thought was that it is not part of the PR, i see that it is, please add info why you changing it into the commit message.
Also if it is not ready for merging, please add "WIP:" prefix to it's name or remove ready for review